frontend/tests/e2e/features/common/index.ts

```diff
@@ -103,13 +103,6 @@ Then('sees the register form', () => {
   cy.screenshot();
 });
 
-Then('sees success messages', () => {
-  cy.get('span.success-message-bottom').should('exist');
-
-  cy.wait(5500);
-  cy.get('span.success-message-bottom').should('not.exist');
-});
-
 Then('sees the create fishbowl form', () => {
```

Comment:
Also, we should check if something here is not "common", like this one.

But that will be for another PR.